.travis.yml

```diff
@@ -14,28 +14,18 @@ notifications:
 env:
   global:
     - MOVEIT_CI_TRAVIS_TIMEOUT=85  # Travis grants us 90 min, but we add a safety margin of 5 min
-    - ROS_DISTRO=melodic
+    - ROS_DISTRO=crystal
     - ROS_REPO=ros
     - UPSTREAM_WORKSPACE=moveit.rosinstall
     - TEST_BLACKLIST=moveit_ros_perception  # mesh_filter_test fails due to broken Mesa OpenGL
     - CXXFLAGS="-Wall -Wextra -Wwrite-strings -Wunreachable-code -Wpointer-arith -Wredundant-decls -Wno-unused-parameter -Wno-unused-but-set-parameter -Wno-unused-function"
     - WARNINGS_OK=false
   matrix:
-    - TEST="clang-format catkin_lint"
-    - TEST=clang-tidy-fix
-    - ROS_DISTRO=melodic
-    - ROS_DISTRO=kinetic
-
-matrix: # Add a separate config to the matrix, using clang as compiler
-  include:
-    # add a config to the matrix using clang as compiler and also test ikfast plugin creation
-    - compiler: clang
-      env: ROS_REPO=ros-shadow-fixed
-           BEFORE_DOCKER_SCRIPT="source moveit_kinematics/test/test_ikfast_plugins.sh"
-           CXXFLAGS="-Wall -Wextra -Wwrite-strings -Wunreachable-code -Wpointer-arith -Wredundant-decls -Wno-unused-parameter -Wno-unused-function -Wno-overloaded-virtual"
```

Comment:
This is some pretty fancy ikfast testing stuff @rhaschke did, at least comment out with TODO to re-instate when ready

Reply:
I believe @rhaschke was the one who first suggested we remove it here. We always have the moveit repo to go back to for examples.

Reply:
There is no reason to remove this. It should compile for MoveIt2 as well, shouldn't it?

Reply:
Restored for now. If it doesn't pass I'll comment out with a todo

Reply:
`panda_ikfast_plugin: Cannot locate rosdep definition for [eigen_conversions]`

Reply:
If you already figured out that eigen_conversions is not available in ROS2 anymore, why not directly replace it with some tf2 equivalents?